lib/Dialect/TTIR/Transforms/Optimizer.cpp

```diff
@@ -0,0 +1,231 @@
+// SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <mlir/Dialect/Func/IR/FuncOps.h>
+#include <mlir/Dialect/Tensor/IR/Tensor.h>
+#include <mlir/Rewrite/FrozenRewritePatternSet.h>
```

Comment:
Are these two rewriter includes really needed?

Reply:
Thanks for the catch, they are not needed. I will remove them!